src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Introduction to app launch times
+tags:
+  - Mobile monitoring
+  - New Relic Mobile
+  - Get started
+metaDescription: Use New Relic's mobile monitoring capabilities to manage the performance of your iOS and Android apps and your customers' overall digital experience.
+---
+
+New Relic's mobile monitoring capabilities help you gain deeper visibility into how to analyze your [Android](/docs/mobile-monitoring/new-relic-mobile-android/get-started/introduction-new-relic-mobile-android) and [iOS](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/introduction-new-relic-mobile-ios) application performance and troubleshoot crashes. With the abundance of high-quality apps out there, users have become very sensitive to app launch time and have very high expectations.
```

Comment:
@ying-nr why would a user need app launch times? what's the goal they're trying to solve by analyzing app launch times? Just trying to get to the heart of the user story.